packages/web/src/components/common/line-graph/LineGraph.tsx

```diff
@@ -120,6 +90,27 @@ const ChartGlobalTooltip = () => {
     />
   );
 };
+function parseTimeTVL(time: string) {
+  const dateObject = new Date(time);
+  const month = dateObject.toLocaleString("en-US", { month: "short", timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone });
+  const day = dateObject.getDate();
+  const year = dateObject.getFullYear();
+  const hours = dateObject.getHours();
+  const minutes = dateObject.getMinutes();
+  const isPM = hours >= 12;
+  const formattedHours = hours % 12 || 12;
+  if (!month || !day || !year)
+  return {
+    date: "Jan 01, 2023",
+    time: "00:00 AM"
```

Comment:
What is the reason for this?

Reply:
Oh sorry. because we had incorrect data yesterday. so i fake this